openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -142,25 +154,46 @@ def handle_notify_credentials(options, course_keys):
             notify_programs=options['notify_programs']
         )
 
-    logger.info('notify_credentials finished')
-
 
 def send_notifications(
-    certs, grades, site_config=None, delay=0, page_size=100, verbose=False, notify_programs=False
+    certs,
+    grades,
+    site_config=None,
+    delay=0,
+    page_size=100,
+    verbose=False,
+    notify_programs=False,
+    revoke_program_certs=False
 ):
-    """ Run actual handler commands for the provided certs and grades. """
+    """
+    Utility function responsible for bootstrapping certificate and grade updates to the Credentials IDA. We do this by
+    invoking existing code paths that should force the LMS to (re-)send data to Credentials.
+
+    First, we directly call our Django signal handlers responsible for updating the status of certificates throughout
+    the system. This will trigger attempts to send certificate status information to the Credentials IDA.
+
+    Then, we call a utility function (`send_grade_if_interesting`) that is responsible for determining if the
+    Credentials IDA cares about this particular grade update for a learner.
+
+    Args:
+        certs (QuerySet): A collection of course certificates associated with the users we need to update
+        grades (QuerySet): A collection of grades associated with the users we need to update
+        site_config (SiteConfiguration): Optional, may be None. The site associated with the users we need to update.
+        delay (int): Delay (in seconds) to wait between queries
+        page_size (int): Number of records to process from the certs or grades QuerySet at once. Used to handle load.
+        verbose (bool): Used to determine level of logging output during grade updates
+        notify_programs (bool): Used to determine if an update should be sent to Credentials
+        revoke_program_certs (bool): Used to determine if the system should attempt revoking program certificates during
+         this run of the `notify_credentials` management command
+    """
     course_cert_info = {}
     # First, do certs
     for i, cert in paged_query(certs, delay, page_size):
         if site_config and not site_config.has_org(cert.course_id.org):
             logger.info("Skipping credential changes %d for certificate %s", i, certstr(cert))
```

Comment:
I take it you're purposefully removing all of the interpolations for `logger.info`, so I figured I'd flag this one. Otherwise, feel free to ignore!